kolibri_explore_plugin/assets/src/modules/topicsTree/handlers.js

```diff
@@ -97,8 +97,7 @@ export function showCustomContent(store, id) {
 
   ConditionalPromise.all(promises).only(
     samePageCheckGenerator(store),
-    // eslint-disable-next-line no-unused-vars
```

Comment:
I realized this lint disabling wasn't needed, just `,  ,` to avoid naming the variable.